gammapy/time/tests/test_exptest.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+from numpy.testing import assert_allclose
+from astropy.units import Quantity
+from gammapy.time.exptest import exptest_for_run
+from gammapy.time import make_random_times_poisson_process
+
+
+def test_exptest_for_run():
+    # setup
+    rate = Quantity(100, 's^-1')
+    time_delta = make_random_times_poisson_process(1000, rate=rate, random_state=0)
+    print('time_delta = {}'.format(time_delta))
```

Comment:
Remove print statements (there's a second one below) from the test.
Also the `setup`, `execution` and `checks on results` comments can be removed.
This was just for explanation ... it's the structure of all tests, but we don't put comments explaining where each section starts, just usually a blank line between those sections.